### ISSUE TITLE
v1.17.2 won't start on macos sierra 10.12.6

### ISSUE THREAD

**Issue by Alexintosh** [Microsoft/vscode#37157]
- VSCode Version:1.17.2
- OS Version: macos sierra 10.12.6

Steps to Reproduce:

1. code . --disable-extensions --verbose

Reproduces without extensions: Yes

Logs from verbose flag:
```
[31276:1030/164811.214784:WARNING:dns_config_service_posix.cc(154)] dns_config has unhandled options!
[main 16:48:11] Sending env to running instance...
[main 16:48:11] Sent env to running instance. Terminating...
```

**Comment by maxibanki** [comment]
Could be a duplicate of #37147. @Alexintosh can you check if in your developer tools the same errors appear?

**Comment by Alexintosh** [comment]
I don't think so @maxibanki, in my case it won't even start.
Although code-insiders will work just fine.

**Comment by octref** [comment]
https://github.com/Microsoft/vscode/issues/35527

Give insiders a try: https://code.visualstudio.com/insiders, some people report it's working for them.

**Comment by Alexintosh** [comment]
@octref yes, I already said in my last comment that insiders works fine. 
This issue shouldn't be closed @octref, it's not the same of #35527.

**Comment by octref** [comment]
Oops, sorry.

So Code does not even start for you? Did you recently upgraded to sierra and it stopped working? Have you tried re-installing?

Meanwhile, does this look like your issue? https://github.com/Microsoft/vscode/issues/19104#issuecomment-276227392
